devito/operator.py

```diff
@@ -90,7 +90,7 @@ class Operator(object):
     def __init__(self, nt, shape, dtype=np.float32, stencils=[],
                  substitutions=[], spc_border=0, time_order=0,
                  forward=True, compiler=None, profile=False,
-                 cache_blocking=False, block_size=5, cb_inner_dim=False,
+                 cache_blocking=False, block_size=5,
```

Comment:
Im gonna fix this in auto tuning  `block_size=None` (default). If block_size is not set. Auto tunning will try to either extract block size from report if one exists otherwise will calculate best square guess based on architecture. @pvelesko equation
Thus `cache_blocking` flag will be necessary.